src/libstd/thread/mod.rs

```diff
@@ -485,15 +485,17 @@ impl Builder {
 /// let (tx, rx) = channel();
 ///
 /// let sender = thread::spawn(move || {
-///     let _ = tx.send("Hello, thread".to_owned());
+///     tx.send("Hello, thread".to_owned())
+///         .expect("Unable to send on channel");
```

Comment:
Invalid indent, should be:

```rust
tx.send("Hello, thread".to_owned())
  .expect("Unable to send on channel");
```

Reply:
> Invalid indent

Huh. [Rustfmt is what indented it this way](https://play.rust-lang.org/?gist=9b83f9fee47ceaa7d3e3338267df36a8&version=stable). Where is this style documented so I can check it before submitting future PRs?

Reply:
rustfmt is really bad... Well, whatever, please align the methods' calls.

Reply:
> Where is this style documented so I can check it before submitting future PRs?

Reply:
No clue. It's how it's done in the rest of rustc source code. Sorry but I have no idea if there is such a document...